src/TodoList.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+const todoList = [
+    {
+        id: 0,
+        title: "Make an appointment",
+    },
+    {
+        id: 1,
+        title: "Finish HW1",
+    },
+    {
+        id:2,
+        title: "Return books"
+    }
+];
+
+
+function TodoList(){
+    return(
+        <ul>
+            {todoList.map(function(item){
+                return(
+                    <li key={item.id}>{item.title}</li>
+                )
+            })}
+        </ul>
+    );
+};
+
+
+export default TodoList;
```

Comment:
nit: github is complaining that there is no newline at the end of the file. You can try adding this extension to VSCode to automatically fix things like this https://prettier.io/